modules/core/src/main/java/org/apache/ignite/internal/IgniteMXBeanImpl.java

```diff
@@ -0,0 +1,341 @@
+    }
+
+    /** {@inheritDoc} */
+    @Override public long getStartTimestamp() {
+        return kernal.startTimestamp();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getStartTimestampFormatted() {
+        return kernal.startTimeFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public long getUpTime() {
+        return kernal.upTime();
+    }
+
+    /** {@inheritDoc} */
+    @Override public long getLongJVMPausesCount() {
+        return kernal.longJVMPausesCount();
+    }
+
+    /** {@inheritDoc} */
+    @Override public long getLongJVMPausesTotalDuration() {
+        return kernal.longJVMPausesTotalDuration();
+    }
+
+    /** {@inheritDoc} */
+    @Override public Map<Long, Long> getLongJVMPauseLastEvents() {
+        return kernal.longJVMPauseLastEvents();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getUpTimeFormatted() {
+        return kernal.upTimeFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getFullVersion() {
+        return kernal.fullVersion();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getCheckpointSpiFormatted() {
+        return kernal.checkpointSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getCurrentCoordinatorFormatted() {
+        return kernal.currentCoordinatorFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public boolean isNodeInBaseline() {
+       return kernal.nodeInBaseline();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getCommunicationSpiFormatted() {
+        return kernal.communicationSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getDeploymentSpiFormatted() {
+        return kernal.deploymentSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getDiscoverySpiFormatted() {
+        return kernal.discoverySpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getEventStorageSpiFormatted() {
+        return kernal.eventStorageSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getCollisionSpiFormatted() {
+        return kernal.collisionSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getFailoverSpiFormatted() {
+        return kernal.failoverSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getLoadBalancingSpiFormatted() {
+        return kernal.loadBalancingSpiFormatted();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getOsInformation() {
+        return kernal.osInformation();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getJdkInformation() {
+        return kernal.jdkInformation();
+    }
+
+    /** {@inheritDoc} */
+    @Override public String getOsUser() {
+        return kernal.osUser();
+    }
+
+    /** {@inheritDoc} */
+    @Override public void printLastErrors() {
+        kernal.context().exceptionRegistry().printErrors(log);
```

Comment:
Lets create `printLastErrors` in the kerrnal like `osUser` methods. Log will not be needed.